test/test/bootstrap.js

```diff
@@ -29,25 +29,30 @@ describe('Bridge', function () {
       const ethAccount = ethClient.accounts[1];
 
       const beforeEthBalance = await ethClient.getDotBalance(ethAccount);
-      const beforeSubBalance = await subClient.queryAccountBalance(polkadotSenderSS58);
+      const beforeSubBalanceAlice = await subClient.queryAccountBalance(polkadotSenderSS58Alice);
+      const beforeSubBalanceBob = await subClient.queryAccountBalance(polkadotSenderSS58Bob);
 
       // lock DOT using basic channel
       await subClient.lockDOT(subClient.alice, ethAccount, amount.toFixed(), ChannelId.BASIC)
+      await subClient.lockDOT(subClient.bob, ethAccount, amount.toFixed(), ChannelId.BASIC)
```

Comment:
A couple of things:

1. There's no guarantee the lockDOT extrinsics for alice and bob will be executed in the same block, and so there may still only be one account per commitment.

I would use the trick with sudo to guarantee we get at least 2 accounts per commitments: `Sudo.sudo(Utility.batch([utility.dispatchAs(Alice, dotApp.lock), ...]))`

Polkadot-JS api docs for utility pallet at https://polkadot.js.org/docs/substrate/extrinsics/#utility

2. I'd feel safer testing with more than 2 accounts. In order to exercise all the proof verification paths a bit more. Can we try 6 or more accounts? These builtin accounts would be: `Alice`, `Bob`, `Charlie`, `Dave`, `Eve`, and [so on](https://github.com/Snowfork/snowbridge/blob/f719545fe8d8cf9b5e0665b690c73077ac49258c/parachain/src/chain_spec/snowbase.rs#L38-L43).

3. Lets rather create a separate test in a new file. This one is purely meant for bootstrapping the bridge (and incidentally testing the basic channel)

Reply:
Cool, I'll set up a separate batch test with at least 6 accounts

Reply:
Will tackle this in [SNO-306](https://linear.app/snowfork/issue/SNO-306)